static/js/components/Select_test.js

```diff
@@ -16,29 +16,34 @@ describe("Select", function() {
     menuPlacement,
     className,
     field,
-    form
+    form,
+    setFieldValueStub
 
   beforeEach(() => {
     sandbox = sinon.createSandbox()
+    setFieldValueStub = sandbox.stub()
     closeMenuOnSelect = false
     openMenuOnClick = false
     isMulti = false
     placeholder = "Select something..."
     menuPlacement = "bottom"
     className = "class"
-    field = { name: "TestName" }
-    form = {}
-    options = [
-      [{ value: 1, label: "Option A" }, { value: 2, label: "Option B" }]
-    ]
+    form = {
+      setFieldValue: setFieldValueStub
+    }
+    ;(options = [
```

Comment:
I think there's some weird formatting stuff here, `;(` and the `( )` around the `field` assignment below